projects/slap_manipulation/configs/language_agent.yaml

```diff
@@ -82,3 +98,188 @@ EVAL_VECTORIZED:
   record_videos: 0          # 1: record videos from printed images, 0: don't
   record_planner_videos: 0  # 1: record planner videos (if record videos), 0: don't
 
+SLAP:
+  dry_run: False
+  min_depth: 0.3
+  max_depth: 1.5
+  x_max: 0.9  # do not care about objects beyond a meter-ish 
+  z_min: 0.15  # 15cm
+  voxel_size_1: 0.001
+  voxel_size_2: 0.01
+  IPM:
+    validate: False
```

Comment:
do we need to document some of these?

Reply:
These are documented in the README. I will add comment based documentations here too.

Reply:
yes add comments here please